TTS/utils/io.py

```diff
@@ -57,7 +56,7 @@ def load_config(config_path: str) -> AttrDict:
 
 
 def load_np_audio_config(config_path):
-    return np.load(config_path, allow_pickle=True).item()['audio_config']
+    return np.load(config_path, allow_pickle=True).item()["audio_config"]
```

Comment:
This is where the tests fail with `Unexpected keyword argument 'allow_pickle' in function call`, but it is a legitimate kwarg and the code works on my side, so I don't know what gives ...

Reply:
https://github.com/PyCQA/pylint/issues/3567

Reply:
I bumped the numpy version up so if you rebase `dev ` then it should pass if the problem is as above .

Reply:
Ok great. Also I'm running Tensorflow 2.4.1 and it seems to be working fine.

Reply:
Did you update the PR? It is still failing I guess.

Reply:
Ok let me try.